### ISSUE TITLE
"LDP Master Builder: All jobs have failed" - problem with ssh key

### ISSUE THREAD

**Issue by pbiering** [tLDP/LDP#108]
https://github.com/tLDP/LDP/actions/runs/3295202869

```
debug1: No more authentication methods to try.
builder@152.19.134.151: Permission denied (publickey).
Error: Process completed with exit code 255.

```

can one fix this?

**Comment by martin-a-brown** [comment]
Hello @ser ,

Do you know how to fix this one?  This looks like an iBiblio relationship thing.

-Martin

```
> dig +noall +answer infra.tldp.ibiblio.org.
infra.tldp.ibiblio.org. 600     IN      A       152.19.134.151
```

**Comment by ser** [comment]
Hello, I'm back from holiday! Checking things out.

**Comment by ser** [comment]
BTW, @martin-a-brown - I was trying to catch you on email before but no response, probably because of spam blocking: "The ubuntu-18.04 environment is deprecated" - and your builder script is failing on 22.04 - are you able to have a quick look into it or you want me to create a VPS with 22.04 for development?

**Comment by ser** [comment]
Guys, the reason of this issue is related to some changes in GitHub actions which do not accept our SSH key in its current format. I'll make a separate test repository to learn why is it happening and will be back here after I understand how to fix it.

**Comment by martin-a-brown** [comment]
Hello Serge,

> BTW, @martin-a-brown - I was trying to catch you on email before but no response, probably because of spam blocking:

Huh.  Must be something like that, as I didn't see anything.  Sorry about that.

> "The ubuntu-18.04 environment is deprecated" - and your builder script is failing on 22.04 - are you able to have a quick look into it or you want me to create a VPS with 22.04 for development?

Right -- this thing hasn't seen a single bit of attention in a few years and I'm sure quite a few things have changed.  I have spun up an Ubuntu 22.04 environment and I've encountered the first of what is likely to be a number of problems whether Python-specific problems (deprecations, versioning, dependencies, libraries) or documentation toolchain or Ubuntu/Debian packaging specific changes.

I've filed the following once I got to the first problem:

https://github.com/tLDP/python-tldp/issues/9

Time to start unraveling the sweater as far as necessary to knit it back into something wearable.

-Martin

**Comment by ser** [comment]
Thanks a lot! Meanwhile I fixed the SSH issues, it was my fault as I did not understand how Actions work and I located the secret key in a wrong place. Now if any of us need write access to the web server, it's possible to use ```SSH_KEY: ${{secrets.IBIBLIO_WWW_SSHKEY}}``` env. It's not available in forks so it should be safe. 

But one thing is still not resolved, I will create a separate issue for this seeking your advice.
